server/internal/usecase/interactor/schema.go

```diff
@@ -91,6 +97,13 @@ func (i Schema) UpdateField(ctx context.Context, param interfaces.UpdateFieldPar
 		if err := updateField(param, f); err != nil {
 			return nil, err
 		}
+
+		if param.IsTitle != nil && *param.IsTitle {
```

Comment:
```suggestion
		if param.IsTitle != nil {
```

Currently there is no way to delete the title field from the schema field.

Reply:
Done!